### ISSUE TITLE
Process documented as a C++11 library

### ISSUE THREAD

**Issue by eldiener** [boostorg/process#172]
The process library should mention in its documentation that it requires at least C++11 support.

**Comment by itss1ddhant** [comment]
@eldiener I am a beginner in open source contribution and would like to take up this issue. Could you help me out with some more details about the issue?

**Comment by eldiener** [comment]
It is not that complicated. The process library is a C++11 library, so somewhere in the documentation for the process library it should mention that it is a C++11 library. The process library requires a compiler at the C++11 standard level or beyond, else it can not be compiled without errors.

**Comment by itss1ddhant** [comment]
Thanks. I am at it.

**Comment by itss1ddhant** [comment]
@eldiener Can you help me with where should I find out the documentation for the Process library. I cloned it but I couldn't find the documentation. Maybe it's just a beginner error!

**Comment by eldiener** [comment]
There is a doc subfolder. The documentation is written in quickbook format. Learn quickbook if you do not know it. It is easy to learn. Are you one of the current maintainers of the process library ?

**Comment by itss1ddhant** [comment]
No I am not a maintainer. And yes, I came across the quickbook .QBK file for process. I was thinking of it too. But I thought it's better to reconfirm. Thanks a lot.

**Comment by eldiener** [comment]
I am glad you are willing to do the work of updating the documentation. Do you know how to create a PR for the work you do ?

**Comment by itss1ddhant** [comment]
Yes , I know how to manage PR and do merge requests but when I will be doing so , or if , I get stuck somewhere in between , do help me. It would greatly add to my learning curve.

**Comment by eldiener** [comment]
No problem. Just ask for help.

**Comment by itss1ddhant** [comment]
I did a fix and generated a PR too but it seems like Travis Cl is continuously building. #174 . Could you see to it or help me out in case of any errors.

**Comment by eldiener** [comment]
The failure occurred because of some previous update on the develop branch. See if you can figure out what previous commit caused the failure. If you can add a fix for the failure to your PR. If you can not you can only try to bring the fatc that a previous update is causing a Travis CI failure to the attention of the process library maintainer.

**Comment by itss1ddhant** [comment]
Okay, I will try to find out that error but in case I fail to do so , how should I know who's the process library maintainer because I see all of the pull requests which have been made so far , even by others in various issues for this repo. have failed to pass through the Travis CI failure.

**Comment by eldiener** [comment]
Leave a note here and hope that the maintainer reads it. As an alternative, or addition, leave a message on the Boost developers mailing list and hope that someone who maintains the process library can fix the problem.

**Comment by itss1ddhant** [comment]
Will do it then. Thanks a lot for the help.
